src/components/BaseSeparatePage/Info/index.tsx

```diff
@@ -13,13 +13,14 @@ export type InfoType = {
   }
   editLink?: string
   className?: string
+  onContributorsButtonClick?: () => void
 }
 
-export const Info: FC<InfoType> = ({ info, editor, editLink, className }) => {
+export const Info: FC<InfoType> = ({ info, editor, editLink, className, onContributorsButtonClick }) => {
   const time = new Date().toDateString()
 
   const ContributorsButton = () => (
-    <button onClick={() => alert('SEE CONTRIBUTORS')} className={styles.contributorsButton}>
+    <button onClick={onContributorsButtonClick} className={styles.contributorsButton}>
```

Comment:
Why not just put the ContributorsDialog inside this component?

Reply:
emmm.... both of them are LGTM. But I thought it would be clearer to make it a child component.